AEPCore/Sources/migration/UserDefaultMigrationConstants.swift

```diff
@@ -177,15 +177,15 @@ enum UserDefaultMigratorConstants {
         static let DATASTORE_NAME = "com.adobe.edge.identity"
         
         enum DataStoreKeys: String, CaseIterable {
-           case IDENTITY_PROPERTIES = "identity.properties"
+            case IDENTITY_PROPERTIES = "identity.properties"
```

Comment:
Did we commit unformatted files earlier, or is there any change in Swift format version that we're using?

Reply:
I'm not too sure how this happened. I just ran swiftformat locally and committed.